Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Receivers/InteractionReceiver.cs

```diff
@@ -41,10 +41,23 @@ public List<GameObject> Interactables
             set { value = interactables; }
         }
 
+        public override bool IsFocusRequired
```

Comment:
Let's be sure to add the summary statement as well ( I know it's kinda redundant, but it's part of our coding guidelines )